src/sonic-py-common/sonic_py_common/general.py

```diff
@@ -43,7 +44,7 @@ def getstatusoutput_noshell(cmd):
     return exitcode, output
 
 
-def getstatusoutput_noshell_pipe(cmd0, *args):
+def getstatusoutput_noshell_pipe(cmd0, *args, return_cmd=True):
```

Comment:
>getstatusoutput_noshell_pipe [](http://example.com/codeflow?start=4&length=28)

Add a unit test for this bug fix?

Reply:
I see you added a unit test. However this PR is fixing something specially for "support displaying output while command is running", not for a quick run command. Is it possible to test it? The test case should fail on current code.